### PR TITLE
Refactor: Lint test/client

### PR DESCRIPTION
**Pull request by mukulljangid**
## High Level Overview of Change
Modify codebase to abide by the ESLint rules.

### Context of Change
Lint Client test files.

### Type of Change

- [X] Refactor (non-breaking change that only restructures code)

## Test Plan
Code works fine when tested against CI test cases.

### CODE REVIEW

File: .eslintrc.js
Comment (mvadari) [inline .eslintrc.js:106]: These rules have been merged in, you should be able to rebase for the updated versions

File: test/client/constructor.ts
Comment (mvadari) [inline test/client/constructor.ts:7]: You can also just ignore the rule here, because you're specifically testing the constructor. I'd do that instead, because it makes the code more readable.

File: test/client/getBalances.ts
Comment (mvadari) [inline test/client/getBalances.ts:16]: This should just be disabled for the whole `test/client` folder

File: test/client/getPaths.ts
Comment (mvadari) [inline test/client/getPaths.ts:1]: If you're deleting these files in this PR, wait until your sugar PR is merged before merging this one.

**Review by mvadari — COMMENTED** [review]
A couple of minor notes

**Comment by mvadari** [comment]
looks like this needs to be rebased

File: .eslintrc.js
Comment (mvadari) [inline .eslintrc.js:84]: This rule should just be turned off for `test/client`

File: test/client/constructor.ts
Comment (mvadari) [inline test/client/constructor.ts:5]: this is backwards - I think the merge got messed up

File: .eslintrc.js
Comment (mvadari) [reply .eslintrc.js:84]: See how rules were turned off for `test/models` as a template

File: test/client/sign.ts
Comment (khancode) [inline test/client/sign.ts:180]: Why is there a `u` added to the end?

**Review by mvadari — APPROVED** [review]

File: test/client/sign.ts
Comment (mukulljangid) [reply test/client/sign.ts]: I added it to fix a linter issue earlier. But, now linter seems to be fine without the `u` as well

File: test/fixtures/requests/getpaths/invalid.json
Comment (mvadari) [inline test/fixtures/requests/getpaths/invalid.json:1]: it's unnecessary to remove these right now - we've got another ticket for dealing with unused fixtures

File: test/client/sign.ts
Comment (mvadari) [reply test/client/sign.ts]: Try refreshing the page - the linter should complain about that

Comment (mvadari) [reply test/client/sign.ts]: Omar - to answer your question: https://eslint.org/docs/rules/require-unicode-regexp

**Review by natenichols — APPROVED** [review]

File: test/client/getOrderbook.ts
Comment (mvadari) [inline test/client/getOrderbook.ts:98]: why was this changed?

**Review by mvadari — APPROVED** [review]

File: test/client/getOrderbook.ts
Comment (mukulljangid) [reply test/client/getOrderbook.ts:98]: changed it back!

**Pull request merged**
